ext/json/config.m4

```diff
@@ -1,6 +1,8 @@
 dnl HAVE_JSON is always 1 as of php 8.0 and the constant will be removed in the future.
 dnl Note that HAVE_JSON was never defined for Windows builds (see config.w32)
 AC_DEFINE([HAVE_JSON],1 ,[whether to enable JavaScript Object Serialization support])
+$RE2C -t ext/json/php_json_scanner_defs.h --no-generation-date -bci -o ext/json/json_scanner.c ext/json/json_scanner.re
+$YACC --defines -l ext/json/json_parser.y -o ext/json/json_parser.tab.c
```

Comment:
If I understand right, this will generate the parser during `./configure` stage, which is not right. The parser should be generated by `make`, and there are already rules for this in `ext/json/Makefile.frag`.

Reply:
Yes, I recompiled and found that the files were generated correctly. It's too strange...
But it seems `that config.w32` generates `json_scanner.c` and `json_parser.tab.c`, and I feel like we can delete it?

Reply:
I'm not familiar with the Windows build system, maybe @cmb69 can check whether or not that code is needed.

Reply:
I don't see why generating the files during configure would be required, since respective make rules are contained in Makefile.frag.w32. So, yes, LGTM.